packages/cli-kit/src/node/cli.ts

```diff
@@ -34,10 +35,12 @@ export async function runCLI(options: RunCLIOptions) {
     settings.debug = true
   } else {
     Bugsnag.start({
+      appType: 'node',
       apiKey: bugsnagApiKey,
       logger: null,
       appVersion: await constants.versions.cliKit(),
       autoTrackSessions: false,
+      autoDetectErrors: false,
```

Comment:
Sorry because I'm probably missing some context, but why do we need this? Where are the uncaught errors coming from? The user code is only run in `dev`, shouldn't it be enough to catch them there?

Reply:
There are errors that go straight to the Node runtime skipping any `try/catch` in our code. [This](https://app.bugsnag.com/shopify/cli/errors/62630ceebd38f1000994a723?filters[release.seen_in]=3.0.27) is an example of one of those errors coming from `esbuild`, which is used internally by `Vite`. I think the reason being is that those tools use [Node thread workers](https://github.com/evanw/esbuild/blob/79a35122062c8e99bf9ab62f03fbafa641139e15/lib/npm/node.ts#L21), which run in their own isolated thread, so Bugsnag is subscribed to those errors and is sending them automatically.

Reply:
Ahh, thanks for the explanation! 👍